assets/sass/helpers/_new_tab_links.scss

```diff
@@ -0,0 +1,27 @@
+// The following classes are used in assets/js/aplus.js. Therefore, you must
```

Comment:
I think this file belongs to the components directory. Helpers are supposed to be for example Sass functions that can be used in other Sass code, but this file really contains styles for certain elements. In other words, it is a component.